packages/react-interactions/events/src/dom/Focus.js

```diff
@@ -36,7 +36,7 @@ type FocusState = {
   isFocused: boolean,
   isFocusVisible: boolean,
   pointerType: PointerType,
-  ...
+  addedRootEvents?: boolean,
```

Comment:
Does it need to be optional?

Reply:
Let's make it just boolean.

Reply:
We don't use it in the Focus responder, only the FocusWithin responder. I didn't want to add more code to the state of that responder, so I opted to keep it optional for now. Breaking all the call-sites down by alternative state would make this a far more complex PR.